client/src/components/previews/LocationPreview.js

```diff
@@ -54,14 +56,26 @@ const LocationPreview = ({ className, uuid }) => {
         <PreviewField label="Name" value={location.name} />
 
         <PreviewField
-          label="Status"
-          value={Location.humanNameOfStatus(location.status)}
+          label="Type"
+          value={Location.humanNameOfType(location.type)}
         />
 
         <PreviewField
-          label="Type"
-          value={Location.humanNameOfType(location.type)}
+          label="Latitude, Longitude"
+          value={`${location.lat}, ${location.lng}`}
```

Comment:
This is not correct; we should show the coordinates in the preferred format set in the dictionary, with the option to switch to a different notation; see the use of `GeoLocation` in **locations/Show.js**.